src/components/services/PiHole.vue

```diff
@@ -37,6 +37,9 @@ export default {
     ads_percentage_today: 0,
   }),
   computed: {
+    apiQuery() {
```

Comment:
Computed property are used to get reactivity in property used in the UI, we don't need that here as this is not displayed. As it's just a simple condition, you can add it to the fetchStatus directly.